lang/contexts.json

```diff
@@ -1,6 +1,7 @@
 {
 	"Paragraph": "Drop-down option label for the paragraph format.",
-	"Heading": "Tooltip and default label for the heading drop-down.",
+	"Heading": "Tooltip for the heading drop-down.",
```

Comment:
#55 is nothing about why tooltip was introduced. And I think it's in the scope of this ticket because we're touching here the label which affects the sense behind a tooltip. But I'll create a separate issue as you wish.